<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyTypesEditor</class>
 <widget class="QDialog" name="PropertyTypesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Property Types Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="propertyTypesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="propertyTypesTable">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="typeDetailsLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="detailsTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
